TestResultSummaryService/package.json

```diff
@@ -17,19 +17,19 @@
   "license": "Apache License Version 2.0",
   "readmeFilename": "README.md",
   "dependencies": {
-    "bluebird": "^3.5.0",
+    "bluebird": "^3.5.3",
     "body-parser": "^1.18.2",
-    "compression": "^1.7.1",
-    "cors": "^2.8.4",
+    "compression": "^1.7.4",
+    "cors": "^2.8.5",
     "csv": "^2.0.0",
-    "express": "^4.15.4",
-    "got": "^9.3.2",
+    "express": "^4.16.4",
+    "got": "^9.6.0",
     "jenkins-api": "^0.3.1",
     "mongodb": "^2.2.31",
-    "request": "^2.83.0",
-    "winston": "^2.4.1"
+    "request": "^2.88.0",
+    "winston": "^2.4.4"
   },
   "devDependencies": {
-    "nodemon": "^1.11.0"
+    "nodemon": "^1.18.10"
```

Comment:
Is upgrade TRSS modules intended in this PR? If so, please make sure TRSS works with upgraded modules.

Reply:
Yeah I didn't see any issues. Since we are only doing minor and no major upgrades, we should be fine technically.

Reply:
Thanks. Just double check as this PR is for PerfNext.